packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -22,19 +22,19 @@ contract L2OutputOracle is OwnableUpgradeable, Semver {
     uint256 public immutable SUBMISSION_INTERVAL;
 
     /**
-     * @notice The number of the first L2 block recorded in this contract.
+     * @notice The time between L2 blocks in seconds.
      */
-    uint256 public immutable STARTING_BLOCK_NUMBER;
+    uint256 public immutable L2_BLOCK_TIME;
 
     /**
-     * @notice The timestamp of the first L2 block recorded in this contract.
+     * @notice The number of the first L2 block recorded in this contract.
      */
-    uint256 public immutable STARTING_TIMESTAMP;
+    uint256 public startingBlockNumber;
```

Comment:
Why is this no longer immutable?

Reply:
I feel like we are just creating churn by changing these from immutables to being in storage

Reply:
It can't be immutable because it's set in the initializer and not in the constructor